### PR TITLE
Added "From zero to Julia!"

### PR DESCRIPTION
**Pull request by aurelio-amerio**
Added "From zero to Julia!", a series of online written lessons about Julia. 
My English is not the best, what I wanted to say is that in this series of lessons you can learn the basics about Julia in a way suitable to a beginner. You will also encounter more advanced topics as you progress. 

Any suggestion on how to edit/improve the description about the course to make it more appealing is welcome! 
Thank you

### CODE REVIEW

File: learning/index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The [Julia Language's YouTube channel](https://www.youtube.com/user/JuliaLanguag
 ## Text
 
 @@tight-list
+* [From zero to Julia!](https://techytok.com/from-zero-to-julia/) by [Aurelio Amerio](https://github.com/aurelio-amerio). An expanding series of short lessons about Julia, starting from beginner level and going up to deal with more advanced topics.
```
Comment (tlienart) [inline learning/index.md:33]: What you've written seems ok, you could maybe go with "from introductory to more advanced topics"? Maybe also swap "lessons" for "tutorials"?

Comment (zdroid) [inline learning/index.md:33]:
```suggestion
* [From zero to Julia!](https://techytok.com/from-zero-to-julia/) by [Aurelio Amerio](https://github.com/aurelio-amerio). An expanding series of short lessons about Julia, starting from the beginner level and going up to deal with the more advanced topics.
```
I think the current description is very good @tlienart. For some reason `from introductory to more advanced topics` sounds more like a description for a book on theoretical mathematics. :smile:
But maybe `tutorials` does sound better than `lessons`.

Comment (tlienart) [reply learning/index.md:33]: 👍

Comment (aurelio-amerio) [reply learning/index.md:33]: @tlienart I like your conciseness and I've swapped lessons with tutorials. Though I wanted to put an emphasis to the fact that it is suitable for people who have little to no knowledge of programming... 
@zdroid I've added the "the"

**Pull request merged**